docs/resources/logbook/2024_lhc.md

```diff
@@ -55,6 +57,7 @@ The tables below can be sorted by clicking next to the column headers.
 *[Squeeze 30cm-22cm]: Squeezing down to 22cm and global corrections with K-mod and checking amplitude detuning.
 *[Measurements with Beam-Beam Wires]: Measured with BB wires at 30cm. Little to no impact seen on coupling and beta-beating. Potential sextupolar RDTs impact. IR5 wire died and we dumped early.
 *[Effect of higher-order contributions to octupole and decapole RDTs]: Test MQ trim. Injection decay measurement. Sensitivity to MCS. Impact of b5 and a4 on the DA and lifetime (LHCB2). MO footprint contraction (LHCB1).
-
+*[Effect of CMS Solenoid on Optics]: Measured with CMS Solenoid at nominal and then ramped down. Measurements at 30cm end-of-MD show a large betabeat degradation compared to final commissioning validation, more than expected. Before / after ramp-down only gives a negigible difference in optics and local coupling. Detuning impact to be checked offline.
+*[HL-LHC Optics Part 1]: Injection optics measured and corrected with success. Ramp was measured and went great except at 1TeV where betabeating goes over 20%. Flat-top optics measured and corrected. Great correction for B2 (incl. dispersion) while B1 needed two knobs (due to exceeding limit of RQTL.9L7.B1), and still shows good control of betas below 10%.
```

Comment:
This summary is not really correct. Beam 1 has only a single knob, Beam 2 has two knobs, but this is not well reflected in the logbook (see my email from that night)